src/lib.rs

```diff
@@ -0,0 +1,10 @@
+pub mod filesystem;
+pub mod manifest;
+pub mod network;
+pub mod ops;
+pub mod path;
+
+pub const COMPANY_NAME: &str = "walnut";
+pub const COMPANY_TLD: &str = "dev";
+pub const MANIFEST_FILE_NAME: &str = "Nargo.toml";
+pub const REPOSITORY_URL: &str = "http://127.0.0.1:8888";
```

Comment:
can we please specify this in ENV? will be easier to manage / build

Reply:
Moved stuff to env